src/common/dns_resolve.cc

```diff
@@ -314,7 +314,7 @@ int DNSResolver::resolve_srv_hosts(CephContext *cct, const string& service_name,
   }
 #endif
   if (len < 0) {
-    lderr(cct) << "res_search() failed" << dendl;
+    lderr(cct) << "res_search() failed" << query_str << dendl;
```

Comment:
@renhwztetecs The change is good. I think you can add a new variable to specify the function name  that was failed also. Create a string like: `res_search() failed for service ` or `res_nsearch() failed for service`. This would be in line (similar) with the previous message before the fix.